README.md

```diff
@@ -198,6 +198,12 @@ password_cmd = "command_that_writes_password_to_stdout"
 # can't be used simultaneously.
 use_keyring = true
 
+#
+# If set to true, `spotifyd` tries to bind to the session dbus
+# and expose MPRIS controls
+#
+use_mpris = false
```

Comment:
i think keeping the current behaviour might be best, so default to using dbus

Reply:
Sorry i didnt explain it correctly, as it currenly is The Default impl of the settings (without a config) will set the value to false. And that would be a breaking change to existing users. I think the best solution would be to make it a `Option<bool>` then when setting the interal config do `config.shared_config.use_mpris.unwrap_or(true)`

Reply:
Looks great!